docs/data_driven_testing.adoc

```diff
@@ -91,88 +91,85 @@ include::{sourcedir}/datadriven/v3/MathSpec.groovy[tag=example]
 
 Let's assume that our implementation of the `max` method has a flaw, and one of the iterations fails:
 
-[source,groovy]
 ----
-maximum of two numbers   FAILED
+maximum of two numbers [a: 1, b: 3, c: 3, #0]   PASSED
+maximum of two numbers [a: 7, b: 4, c: 7, #1]   FAILED
 
 Condition not satisfied:
 
 Math.max(a, b) == c
-    |    |  |  |  |
-    |    7  4  |  7
-    42         false
+|    |   |  |  |  |
+|    |   7  4  |  7
+|    42        false
+class java.lang.Math
+
+maximum of two numbers [a: 0, b: 0, c: 0, #2]   PASSED
 ----
 
 The obvious question is: Which iteration failed, and what are its data values? In our example, it isn't hard to figure
-out that it's the second iteration that failed. At other times this can be more difficult or even impossible.
-footnote:[For example, a feature method could use data variables in its `given:` block, but not in any conditions.]
-In any case, it would be nice if Spock made it loud and clear which iteration failed, rather than just reporting the
-failure. This is the purpose of the `@Unroll` annotation.
+out that it's the second iteration (with index 1) that failed even from the rich condition rendering. At other times
+this can be more difficult or even impossible.footnote:[For example, a feature method could use data variables in its
+`given:` block, but not in any conditions.] In any case, Spock makes it loud and clear which iteration failed, rather
+than just reporting the failure. Iterations of a feature method are by default unrolled with a rich naming pattern.
+This pattern can also be configured as documented at <<Unrolled Iteration Names>> or the unrolling can be disabled
+like described in the following section.
 
-== Method Unrolling
+== Method Rolling and Unrolling
```

Comment:
IMHO `roll` is not the inverse of `unroll`, `rollup` would be the inverse. Think about a carpet, if you start with it flat on the ground you can roll it up, then you can roll it around (it is still a roll nothing changes) and unroll it again.

Reply:
I partly agree, that's why I called it `@Rollup` and not `@Roll` like in szpaks extension.
While probably correcter would be `@RollUp`.
But `Method Rollupping` sounds strange. or would it then be `Uprolling`?
Do we have a native speaker around that can help with the phrasing?

Reply:
Nice analogy to a carpet :)

Reply:
https://en.wiktionary.org/wiki/uprolling#English and
https://www.merriam-webster.com/dictionary/uproll
seems correct